src/components/Button/Button.tsx

```diff
@@ -52,6 +53,7 @@ const Button: FC<ButtonComponentProps> = ({
                         {children}
                     </IconButton>
                 </ButtonComponent>
+                <div>{iconLabel}</div>
```

Comment:
I couple changes regarding the logic:
1) Can you add a class-name to this div (like `icon-label`) so that developers can more easily target and customize it should they need to.
2) I think the div should not render if there is no text.. i.e. `{iconLabel && <div>{iconLabel}</div>}`
3) For styling changes, please add a little more margin above the text (so there's some space when the button has a solid primary/background) and also center the text beneath the button. This might require an additional parent wrapper that contains and aligns both the button and the iconLabel --- which could also be optionally existing/styled based on if iconLabel is present or not).

Current
![image](https://user-images.githubusercontent.com/80778550/210588544-69ad5e88-d327-4f09-8315-9d27ac9c66e1.png) 
![image](https://user-images.githubusercontent.com/80778550/210588707-1613de33-693e-4172-916f-445161b6352f.png)



Preferred
![image](https://user-images.githubusercontent.com/80778550/210588404-737297b2-c750-4f5e-a41f-60bae935c048.png) 
![image](https://user-images.githubusercontent.com/80778550/210588843-3992936f-0899-479e-983f-b5841dde5c53.png)